src/languages/lua.coffee

```diff
@@ -19,4 +19,9 @@ module.exports = {
 
   defaultBeautifier: "Lua beautifier"
 
+  options:
+    do_not_change_whitespace:
+      type: 'boolean'
+      default: false
+      description: "do not adjust whitespace"
```

Comment:
- [ ] Capitalize `do` (to `Do`) so it does not look unprofessional
- [ ] Consider improving the description. What does "adjust whitespace" mean? Why would a user not want to do it?